cluster/manifests/zmon-agent/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: "zmon-agent"
+  namespace: "kube-system"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        application: "zmon-agent"
+        version: "v0.1"
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+        scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
+    spec:
+      containers:
+        - name: zmon-agent
+          image: "registry.opensource.zalan.do/zmon/zmon-agent-core:0.1-a15"
+          env:
+            - name: ZMON_AGENT_INFRASTRUCTURE_ACCOUNT
+              value: "{{ .InfrastructureAccount }}"
+            - name: ZMON_AGENT_ACCOUNT_ALIAS
+              value: "{{ .ConfigItems.account_alias }}"
+            - name: ZMON_AGENT_REGION
+              value: "{{ .Region }}"
+            - name: ZMON_AGENT_INTERVAL
+              value: "60"
+
+            - name: ZMON_AGENT_ENTITY_SERVICE_URL
+              value: {{ .ConfigItems.zmon_url }}
+
+            - name: ZMON_AGENT_KUBERNETES_CLUSTER_ID
+              value: "{{ .ID }}"
+            - name: ZMON_AGENT_KUBERNETES_UPDATE_CERTIFICATES
+              value: "true"
```

Comment:
hmm, it should definitely work without it (it will use the `ca.crt` for requests' `verify` kwarg)